jenkins/envs/dom.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+source $(dirname "$BASH_SOURCE")/base.sh
```

Comment:
Maybe we could also source the Daint env and just override the required settings? As far as I can tell there is only `GTRUN_SBATCH_ACCOUNT` and `GTRUN_SBATCH_PARTITION` that differ from the Daint setup, right? This would also include the `module load` return code fix then.

Reply:
The module load return code fix doesn't work on dom. That's the reason why I switched to copying the whole file.

Reply:
Doesn't work = makes `module` command a noop. I didn't want to investigate further.